pkg/api/testapi/testapi_test.go

```diff
@@ -46,59 +41,9 @@ func TestResourcePathWithPrefixForV1Beta3(t *testing.T) {
 	}
 }
 
-func TestResourcePathWithPrefixForV1(t *testing.T) {
-	if Version() != "v1" {
-		// Skip the test if we are not testing v1.
-		return
-	}
-
-	testCases := []struct {
-		prefix    string
-		resource  string
-		namespace string
-		name      string
-		expected  string
-	}{
-		{"prefix", "resource", "mynamespace", "myresource", "/api/v1/prefix/namespaces/mynamespace/resource/myresource"},
-		{"prefix", "resource", "", "myresource", "/api/v1/prefix/resource/myresource"},
-		{"prefix", "resource", "mynamespace", "", "/api/v1/prefix/namespaces/mynamespace/resource"},
-		{"prefix", "resource", "", "", "/api/v1/prefix/resource"},
-		{"", "resource", "mynamespace", "myresource", "/api/v1/namespaces/mynamespace/resource/myresource"},
-	}
-	for _, item := range testCases {
-		if actual := ResourcePathWithPrefix(item.prefix, item.resource, item.namespace, item.name); actual != item.expected {
-			t.Errorf("Expected: %s, got: %s for prefix: %s, resource: %s, namespace: %s and name: %s", item.expected, actual, item.prefix, item.resource, item.namespace, item.name)
-		}
-	}
-}
-
-func TestResourcePathForV1Beta3(t *testing.T) {
-	if Version() != "v1beta3" {
-		// Skip the test if we are not testing v1beta3.
-		return
-	}
-
-	testCases := []struct {
-		resource  string
-		namespace string
-		name      string
-		expected  string
-	}{
-		{"resource", "mynamespace", "myresource", "/api/v1beta3/namespaces/mynamespace/resource/myresource"},
-		{"resource", "", "myresource", "/api/v1beta3/resource/myresource"},
-		{"resource", "mynamespace", "", "/api/v1beta3/namespaces/mynamespace/resource"},
-		{"resource", "", "", "/api/v1beta3/resource"},
-	}
-	for _, item := range testCases {
-		if actual := ResourcePath(item.resource, item.namespace, item.name); actual != item.expected {
-			t.Errorf("Expected: %s, got: %s for resource: %s, namespace: %s and name: %s", item.expected, actual, item.resource, item.namespace, item.name)
-		}
-	}
-}
-
-func TestResourcePathForV1(t *testing.T) {
-	if Version() != "v1" {
-		// Skip the test if we are not testing v1.
+func TestResourcePath(t *testing.T) {
+	if Version() != ""+Version()+"" {
```

Comment:
A wrong "find and replace". Maybe we can just use my PR #9263 if you are okay with it.